src/_pytest/fixtures.py

```diff
@@ -1352,7 +1360,7 @@ def fixture(
 def yield_fixture(
     fixture_function=None,
     *args,
-    scope="function",
+    scope: _ScopeName = "function",
```

Comment:
`yield_fixture` is a legacy thing and is intentionally not typed, so I would omit this change.

Reply:
This comment is still relevant